### PR TITLE
Improving chinese tokenizer

### PR DESCRIPTION
**Pull request by wangfeng-skymind**
The source code of the open source algorithm has been integrated into the dl4j, and the source code contains  the original function comment and contributor names that wasn't deleted yet,Im not sure whether deleting the comment and name is a way of good,so I didn't do
(Please fill in changes proposed in this fix)

The patch has passed the test

### CODE REVIEW

File: deeplearning4j-nlp-parent/deeplearning4j-nlp-chinese/src/main/java/org/apdplat/word/WordFrequencyStatistics.java [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.apdplat.word;
```
Comment (raver119) [inline deeplearning4j-nlp-parent/deeplearning4j-nlp-chinese/src/main/java/org/apdplat/word/WordFrequencyStatistics.java]: This thing states GPL in header. Isn't that a blocker? cc @agibsonccc @huitseeker

**Comment by agibsonccc** [comment]
@wangfeng-skymind could you handle the conflicts please?

File: deeplearning4j-nlp-parent/deeplearning4j-nlp-chinese/pom.xml [hunk 2]
```diff
@@ -25,10 +37,18 @@
...
         </dependency>
         <dependency>
-            <groupId>org.apdplat</groupId>
-            <artifactId>word</artifactId>
-            <version>1.3</version>
+            <groupId>org.nutz</groupId>
+            <artifactId>nutz</artifactId>
+            <version>1.r.58</version>
+            <scope>provided</scope>
+        </dependency>
```
Comment (agibsonccc) [inline deeplearning4j-nlp-parent/deeplearning4j-nlp-chinese/pom.xml]: This provided scope doesn't make sense..

**Comment by wangfeng-skymind** [comment]
Already been handled,

**Pull request merged**